docs/DeveloperGuide.adoc

```diff
@@ -782,7 +783,14 @@ Use case ends.
 * 2a. Person already exists in AddressBook.
 +
 [none]
-** 2a1. AddressBook shows an error message.
+** 2a1. AddressBook shows an error message indicating person already exists.
++
+Use case ends.
+
+* 2b. Person added is of invalid format.
++
+[none]
+** 2b1. AddressBook shows an error message indicating person entered is of invalid format.
```

Comment:
I think that the app would display help in case of failure. Moreover, still to my mind it is not ok to have the extensions of 2nd instead of 1st point. I think that textbook follows different convention.

Reply:
It shows an error message together with help. Yeap I have a think about it, should be under 1. Thanks for pointing out.